qiita_ware/ebi.py

```diff
@@ -20,6 +21,11 @@
 from qiita_db.util import convert_to_id
 
 
+class EmptyFileWarning(Warning):
```

Comment:
I'll move the warning to `qiita_ware.exceptions` and import it here. What is the purpose of the filter? i.e., in what circumstances would the warning not be raised if the `warnings.simplefilter("always")` were not there?